arangod/VocBase/vocbase.h

```diff
@@ -139,6 +139,7 @@ struct TRI_vocbase_t {
   enum class State { NORMAL = 0, SHUTDOWN_COMPACTOR = 1, SHUTDOWN_CLEANUP = 2 };
 
   TRI_vocbase_t(TRI_vocbase_type_e type, arangodb::CreateDatabaseInfo const&);
+  TRI_vocbase_t(TRI_vocbase_type_e type, arangodb::CreateDatabaseInfo&&);
```

Comment:
is this constructor actually ever used?
I don't see any callers in this PR, and the old code seemed to have got along without it as well.
So if we don't absolutely need it, I would not add it, given the added complexity it brings.